app/controllers/ansible_roles_controller.rb

```diff
@@ -5,10 +5,11 @@ class AnsibleRolesController < ::ApplicationController
   before_action :find_resource, :only => [:destroy]
   before_action :find_proxy, :only => [:import]
   before_action :create_importer, :only => [:import, :confirm_import]
+  before_action :default_order, :only => [:index]
 
   def index
     @ansible_roles = resource_base.search_for(params[:search],
-                                              :order => params[:order]).
+                                              :order => ()).
```

Comment:
Shouldn't this still be `params[:order]` in order to use `default_order`?

Reply:
Yes, it should. fixed.